components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -29,3 +29,6 @@ export const getRampNetworkPrefix = (chainId: string) => {
       return ''
   }
 }
+
+export const toDouble = (value: string) =>
```

Comment:
Maybe rename to something like `formatAsDouble` since it returns a string?

Reply:
Done!